script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -11,7 +11,7 @@ My arm feels a little heavy as I reach out for my glasses.
 [sha:01a4dc8776eae4101c088541ad958aeb201c83b9]{
 -- Page 1, Offset 16567.
 -- ああ、これは微熱があるな、と他人事のように思いながら、眼鏡をかける。
-Ah, I must have developed a light fever. With that detached thought, I put them on.
+Ah, I must have developed a slight fever. With that detached thought, I put them on.
```

Comment:
revert

Reply:
isn't slight fever the correct term? we can use low-grade if we want to be more technical

Reply:
ngram agrees with you but "light fever" sounds right to me. I'll let @rschlaikjer decide

Reply:
sure

Reply:
Light fever is what felt right to me too